Brokerages/Alpaca/AlpacaBrokerage.Utility.cs

```diff
@@ -417,6 +414,8 @@ private static Holding ConvertHolding(IPosition position)
                 Symbol = symbol,
                 Type = securityType,
                 AveragePrice = position.AverageEntryPrice,
+                MarketPrice = position.AssetCurrentPrice,
```

Comment:
Should this be `position.AssetLastPrice`? It seems that `position.AssetCurrentPrice` has a value of zero

Reply:
@StefanoRaggi position.current_price in REST API should not be 0. Which environment/symbol are you seeing it?

Reply:
@umitanuki I noticed it this morning with an existing SPY position on a paper account

Reply:
@StefanoRaggi @umitanuki - maybe it happened because SPY was out of trading session? We can try `position.AssetCurrentPrice` first and if it equal to 0 fallback to `position.AssetLastPrice` value. Will it work?

Reply:
@OlegRa When I tested it, the equity market was open, it was 10:44 NewYork time:
```
2019-04-22T14:44:07.8367573Z TRACE:: BrokerageSetupHandler.Setup(): Has existing holding: SPY: 53 @ $278.19 - Market: $0 - Conversion: 
...
2019-04-22T14:44:16.0398039Z TRACE:: Log: 4/22/2019 10:44:11 AM - SPY - 289.7403
2019-04-22T14:44:16.0416610Z TRACE:: Log: 4/22/2019 10:44:12 AM - SPY - 289.7479
2019-04-22T14:44:16.0426642Z TRACE:: Log: 4/22/2019 10:44:13 AM - SPY - 289.7479
2019-04-22T14:44:16.0426642Z TRACE:: Log: 4/22/2019 10:44:14 AM - SPY - 289.73
```
Stepping through it with the debugger I verified that `position.AssetLastPrice` was a valid price and `position.AssetLastPrice` was zero.

Reply:
@StefanoRaggi @umitanuki I've found problem - it's already fixed in original SDK sources many months ago and I just forgot about this fix. Will update PR soon.

Reply:
@StefanoRaggi Fixed